quodlibet/quodlibet/images/hicolor/Makefile

```diff
@@ -0,0 +1,82 @@
+QL_SOURCE=scalable/apps/quodlibet.svg
+QL_DEST=quodlibet.png
+
+QL_SYM_SOURCE=scalable/apps/quodlibet-symbolic.svg
```

Comment:
The "-symbolic" icons can't be converted to png. They are a special svg variant that allows color rewriting.